flocker/provision/_install.py

```diff
@@ -517,48 +516,48 @@ def task_open_control_firewall(distribution):
 
 
 def task_enable_flocker_agent(distribution, control_node,
```

Comment:
Now our tests are testing something different to the docs. I think that this would perhaps be split up into two separate tasks:
* One which creates an `agent.yml`.
* One which starts the agents.

This would allow:
* All the functionality to be in the docs and tested.
* The docs to use the former task in the ZFS section above where these commands are rendered, in place perhaps of the example YAML.
* The docs to use the former task with different dataset arguments (maybe not now) for each other backend to allow easy testing of the docs.

I like having the `cat <<` command just because it stops someone having to write that themselves.